README.md

```diff
@@ -25,13 +25,52 @@ And 4.2.14+amd64 would only apply to the amd64 release image.
 ### Add Releases To Channels
 
 Edit the appropriate file in `channels/`.
-For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`.
-Channel semantics are documented [here][channel-semantics].
+For example, to add a release to candidate-4.2 you would edit [`channels/candidate-4.2.yaml`](channels/candidate-4.2.yaml).
 
 The file contains a list of versions.
 Please keep the versions in order.
 And LEAVE COMMENTS if you skip a version.
 
+#### Feeder Channels
+
+Channel semantics, as documented [here][channel-semantics], show nodes and edges being promoted to successive channels as they prove their stability.
+For example, a 4.2.z release will appear in `candidate-4.2` first.
+Upon proving itself sufficiently stable in the candidate channel, it will be promoted into `fast-4.2`.
+Some time after landing in `fast-4.2`, it will appear in `stable-4.2`.
+
+_Note:_ Once we have phased release rollouts, we will drop the fast/stable distinction from this repository and promote to a unified fast/stable channel with a start time and rollout duration.
+Until then, we are using fast channels to feed stable channels with a delay, just like candidate channels feed fast channels.
+
+In this repository, the intended promotion flow is reflected by a `feeder` property in the channel declaration.
+For example, for [`channels/fast-4.2.yaml`](channels/fast-4.2.yaml):
+
+```yaml
+feeder:
+  name: candidate-4.2
+  delay: PT24H
+```
+
+which declares the intention that nodes and edges will be considered for promotion into `fast-4.2` after cooking for 24 hours in `candidate-4.2`.
+The `delay` value is an [ISO 8601][rfc-3339-p13] [duration][iso-8601-durations].
+
+This is the expected delay, but it does not mean that promotion will happen at that moment.
```

Comment:
By which automation is this delay used and how would one override it?

Reply:
> By which automation is this delay used...

By a periodic that runs `stabilization-changes` and creates PRs for any recommended promotions.  Then eventually, when we trust our automatic edge-pulling, we give the robot creds to approve their own PRs so they will auto-merge.

> ... and how would one override it?

Manually create pull requests if you don't want to wait the configured delay?

Reply:
If / when robots were auto-merging these PRs, how would release admins override the bot from promoting something

Reply:
The most important action is ART publishing the errata, which is when a release is declared supported, regardless of what we do in Cincinnati.  So you could tombstone a release if you didn't want it auto-promoted.  But if the release is only in candidate, you had better also tell ART to not ship the errata.  And if the errata has shipped, tombstoning fast or stable promotions won't drop any supported-ness.